vulnfeeds/git/repository.go

```diff
@@ -63,6 +69,29 @@ func RepoTags(repoURL string, repoTagsCache *RepoTagsMap) (versions Versions, e
 	return versions, nil
 }
 
+// NormalizeRepoTags populates a durable mapping of tags to Versions for lookup by normalized tag.
+func NormalizeRepoTags(repoURL string, normalizedRepoTags NormalizedRepoTagsMap, repoTagsCache *RepoTagsMap) (e error) {
```

Comment:
it's not clear what's the output of this function, as it's the second argument here. Can we return the NormalizedRepoTagsMap map instead?

Reply:
I mean the bits that did get computed (not NormalizedRepoTagsMap as a whole): 

i.e. either the `map[string]Version` as it is now or `NormalizedVersions` as I suggested above with the refactoring.

Reply:
Yep, done in [6907948](https://github.com/google/osv.dev/pull/1057/commits/69079488e660ad00d4d6971eb2bb36ca0a7dc42c)